src/Microsoft.DotNet.Tools.New/FSharp_Console/project.json

```diff
@@ -1,4 +1,4 @@
-﻿{
+{
     "version": "1.0.0-*",
     "compilationOptions": {
         "emitEntryPoint": true
```

Comment:
What about this file - https://github.com/dotnet/cli/pull/1077/files#diff-e933ad30dec8519bdbbe45d501906a85R14?